rules/network/discovery_post_exploitation_public_ip_reconnaissance.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/09/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+It is common for adversaries to test for Internet access and acquire their public IP address after they have gained access to a system. Among others, this has been observed in
+campaigns leveraging the information stealer, Trickbot. This rule identifies domains commonly used by adversaries for
+post-exploitation IP reconnaissance.
```

Comment:
Just a friendly reminder to lint this rule before it's merged.

Reply:
Done. Sorry 'bout that.